examples/stm32/f3/nucleo-f303re/miniblink/miniblink.c

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
+ * Copyright (C) 2011 Stephen Caudle <scaudle@doceme.com>
+ * Modified by Fernando Cortes <fermando.corcam@gmail.com>
+ * modified by Guillermo Rivera <memogrg@gmail.com>
+ * Modified by Josh Myer <josh@joshisanerd.com>
```

Comment:
I'm unconvinced they have copyright on this file either, but this is an area where you could argue it.